Makefile.include

```diff
@@ -130,6 +130,14 @@ endif
 include $(RIOTBASE)/Makefile.modules
 include $(RIOTBOARD)/$(BOARD)/Makefile.include
 include $(RIOTCPU)/$(CPU)/Makefile.include
+
+# get number of interfaces straight before resolving dependencies
+NG_NETIF_NUMOF ?= 1
+
+ifneq ($(NG_NETIF_NUMOF),1)
+  CFLAGS += -DNG_NETIF_NUMOF=$(NG_NETIF_NUMOF)
+endif
```

Comment:
Hm, this way a netif driver cannot make a `NG_NETIF_NUMOF++`. I don't know if this is a sensible use case.